City-of-Pori/pori-kada
# PORI-120 Liftup fixes

Fixes issues with topic and project liftup styles being messed up in the master branch.
To test:

1. Sync database from production. If this is not possible add the following items to your front page:
- at least four topics with at least one topic with image and one without image
- at least two project liftups with reasonable amount of text and a title
2. drush cc all
3. Go to front page and make sure that the project liftup and topics look as in the layout and scale nicely. Layouts can be found from here: 
- https://drive.google.com/drive/u/0/folders/0B7ZHsug7PcEDVDBNTG51a0RIMzQ (project mobile)
- https://drive.google.com/drive/u/0/folders/0B7ZHsug7PcEDVDBNTG51a0RIMzQ (page 3: project desktop, page 5 topics desktop)
